tests/jeos/sccreg.pm

```diff
@@ -26,7 +26,7 @@ sub run {
     assert_script_run "SUSEConnect --url=$url -r $scccode";
     assert_script_run 'SUSEConnect --list-extensions';
     # Install Server-like modules
-    for my $module ('basesystem', 'scripting', 'legacy') {
+    for my $module ('basesystem', 'scripting', 'desktop-applications', 'server-applications', 'development-tools', 'legacy') {
```

Comment:
Maybe you can now use the same approach as in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/3776/files . So could be something like:

```perl
    foreach ((split(',', $registration::SLE15_DEFAULT_MODULES{get_required_var('SLE_PRODUCT')}), qw(sdk legacy)) { …
```

but we can also leave it explicit for now. Your choice.

Reply:
I put in a variant on that in `console/suseconnnect_scc` test which I can now use use in JeOS. @yxususe let me know if the addition is fine with you.